web/teamail/build.js

```diff
@@ -0,0 +1,10 @@
+const serverConfig = {
+  servedir: __dirname,
+  port: 3000
+}
+
+require('esbuild').serve(serverConfig, {
```

Comment:
Nit: We'll probably also want --target=esnext --sourcemap=external. The sourcemap will be a hard requirement for debugging I think.